examples/cloud_run_cloudevents/README.md

```diff
@@ -0,0 +1,23 @@
+# Deploying a CloudEvent Function to Cloud Run with the Functions Framework
+
+This sample uses the [CloudEvents SDK](https://github.com/cloudevents/sdk-python) to send and receive a CloudEvent on Cloud Run.
+
+## How to run this locally
+
+Build the Docker image:
+
+```commandline
+docker build -t ff_example .
+```
+
+Run the image and bind the correct ports:
+
+```commandline
+docker run --rm -p 8080:8080 -e PORT=8080 ff_example
+```
+
+Send an event to the container:
+
+```python
+python send_cloudevent.py
+```
```

Comment:
What will this print out? I'd expect something like:

```
We'll see "Hello, World" in stdout.
```

Reply:
send_cloudevent.py in itself doesn't print anything. The functions-framework server innstance however will print out:
```python
print(f"Received event with ID: {cloudevent['id']} and data {cloudevent.data}")
```

Reply:
Lmk if you'd want me to change that @grant

Reply:
I raised a concern with cloudevent.data in reference to the error messages that would hold for this too. Clarifying that this needs to be changed and if so changing it is my only blocker.

Reply:
Not printing anything is fine. This looks good to me.